docs/src/main/asciidoc/kafka.adoc

```diff
@@ -2052,7 +2053,75 @@ class BaristaTest {
 <3> Retrieve the outgoing channel (`beverages`) - the channel must have been switched to in-memory in the test resource.
 <4> Use the `send` method to send a message to the `orders` channel.
 The application will process this message and send a message to `beverages` channel.
-<5> Use the `received` method on `beverages` channel to check the messages produced by the application.
+<5> Use the `received` method on `beverages` channel to check the messages produced by the application. The `await` is imported from https://github.com/awaitility/awaitility[awaitility].
+
+If your Kafka consumer is batch based, you will need to send a batch of messages to the channel as per discussed in this https://github.com/smallrye/smallrye-reactive-messaging/discussions/1691#discussioncomment-2471033[Github Discussion].
```

Comment:
I don't think we should link to a discussion. Our doc should be authoritative.

Reply:
done